### DISCUSSION TITLE
Why KafkaFetcherConfig createTopicIfNotExist is not used?

### DISCUSSION THREAD

**Discussion by tsonglew** [apache/skywalking#12094] [Q&A] [answered]
I want to skip some topics when using kafka-fetcher, and I thought KafkaFetcherConfig.createTopicIfNotExist is what I need. But it seems that this property is not used. Is this by design?

https://github.com/apache/skywalking/blob/13bb3f62a6ff809c148dbeb3f2a9575fbcd71f17/oap-server/server-fetcher-plugin/kafka-fetcher-plugin/src/main/java/org/apache/skywalking/oap/server/analyzer/agent/kafka/module/KafkaFetcherConfig.java#L48

**Comment by wu-sheng** [comment]
I think this is legacy codes, it should be removed. Current codes should create topics once they are missing.

**Reply by wu-sheng** [reply, marked as answer]
This was abandoned in #7326 (v8.7.0)
